nucliadb_utils/nucliadb_utils/grpc.py

```diff
@@ -23,7 +23,9 @@
 
 from grpc import aio  # type: ignore
 from grpc import ChannelCredentials
+
 from nucliadb_telemetry.grpc import GRPCTelemetry
+from nucliadb_telemetry.grpc_sentry import SentryInterceptor
```

Comment:
we'll need a new release of the nucliadb-telemetry package first in practice so we will need to split this into a separate PR.